app/models/like.rb

```diff
@@ -0,0 +1,6 @@
+class Like < ApplicationRecord
+  belongs_to :post
+  def update_likes_counter_for_user
+    post.update(likes_counter: post.likes.count)
+  end
+end
```

Comment:
- [ ] Like belongs to a user. Please make sure to set up an association between like and author(user) on this model.

Reply:
Hi @thecodechaser you right, I did fix it